### PR TITLE
[13.0][FIX] mass_operation_abstract: change translations variables formatting

### PR DESCRIPTION
**Pull request by ghost**
Change translations, in mass_operation_wizard_mixin and mass_operation_mixi, from _("%d" % variable) to _("%d") % variable as specified in odoo doc.

Also added geman translations for some terms.

### CODE REVIEW

**Review by BT-cserra — APPROVED** [review]

**Comment by BT-cserra** [comment]
(Didn't check the German translations, but I know they come from German-speaking people)

**Review by BT-cmorales — APPROVED** [review]
LGTM! All I see is coherent :thumbsup:

**Comment by ghost** [comment]
@LoisRForgeFlow Could you check this pr correctness and flows? . (I not you are not the maintainer of this module but you are of base_tier_validation that use this one). Thx in advance :smile:.

**Comment by LoisRForgeFlow** [comment]
@BT-prenero this is not a dependency of tier validation modules but I can review anyway :wink:

File: mass_operation_abstract/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Translation of Odoo Server.
```
Comment (LoisRForgeFlow) [inline mass_operation_abstract/i18n/de.po:1]: Ideally translations should be done using transifex.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, the translations should have been done using transifex afterward, but I understand that since you are fixing the tranlated variables you need to do it in the pr.

File: mass_operation_abstract/i18n/de.po (hunk 1 above)
Comment (ghost) [reply mass_operation_abstract/i18n/de.po:1]: Sorry. I did not know about that. I see OCA also have a tool called weblate. In future situations Ill do it that way. Thx for your review :smile:.

Comment (LoisRForgeFlow) [reply mass_operation_abstract/i18n/de.po:1]: Exactly, weblate is the new tool! Thanks for contributing :)

**Comment by LoisRForgeFlow** [comment]
@BT-prenero You need to run pre-commit, you can see it is failing now https://github.com/OCA/server-ux/runs/4239399881?check_suite_focus=true. When you fix it, I will run the merge command again.

**Comment by ghost** [comment]
> @BT-prenero You need to run pre-commit, you can see it is failing now https://github.com/OCA/server-ux/runs/4239399881?check_suite_focus=true. When you fix it, I will run the merge command again.

I did not found the way to set them in local for a contribution. I only found then for migration. Is there any link where I can check how to set that precommits in my local repo ?

**Comment by ghost** [comment]
@LoisRForgeFlow I just updated the code after run the pre-commit (i think this time I did it correctly). Pls let me know if I miss something. Thx for all.

**Comment by LoisRForgeFlow** [comment]
Hey, thanks! travis is still failing, can you check?

**Comment by ghost** [comment]
@LoisRForgeFlow seems to be a setupclass failing from this test file `odoo.addons.document_quick_access_folder_auto_classification.tests.test_document_quick_access_auto_classification.TestDocumentQuickAccessClassification`. Nothing to do with what I did change. But I could check it later.

**Comment by LoisRForgeFlow** [comment]
Try to rebase your branch, there have been recent commits in that module, also, please squash your commits into one.

Thanks!

**Comment by ghost** [comment]
> Try to rebase your branch, there have been recent commits in that module, also, please squash your commits into one.
> 
> Thanks!

@LoisRForgeFlow done. Lets see what is the result from travis.

**Comment by LoisRForgeFlow** [comment]
Fixing travis issue here: https://github.com/OCA/server-ux/pull/408.

**Comment by ghost** [comment]
@LoisRForgeFlow I see travis still has the same issue outside my change. What is the way to go now ?

**Comment by LoisRForgeFlow** [comment]
The merge branch is fine... let's try once more:

/ocabot merge patch

**Pull request merged**